packages/api/tests/endpoints/test-granules.js

```diff
@@ -446,6 +447,48 @@ test.serial('default lists and paginates correctly with search_after', async (t)
   t.not(meta.searchContext === newMeta.searchContext);
 });
 
+test.serial('default lists and paginates correctly', async (t) => {
```

Comment:
Ok, so now I get that this is the only test where the search will use the condition and go to RDS, and all the other tests for the list endpoint will go to ES. And I know that these tests will probably change a lot very soon. But from not having worked on it, it's not clear to me that this test goes to RDS and the others to ES, so maybe for now, we can have a comment for this test saying that this is the one that will end up querying RDS? Or maybe in the test description, but just some way of making the distinction clear.